package.json

```diff
@@ -39,11 +39,13 @@
   },
   "scripts": {
     "bootstrap:hoist": "lerna bootstrap --hoist --nohoist=one-cerner-style-icons",
-    "clean": "lerna clean --yes && rm -rf node_modules || true",
-    "clean:install": "npm run clean && npm install",
+    "clean": "npm run clean:lib && npm run clean:root",
+    "clean:install": "npm run clean:all && time npm install",
+    "clean:lib": "rm -rf packages/**/lib",
+    "clean:root": "$(lerna clean --yes || true) && echo 'removing ./node_modules' && rm -rf node_modules && echo 'removed ./node_modules'",
     "clean:obsolete-snapshots": "npm test -- -u",
     "create-props-markdown": "node scripts/props-table-gen/generatePropsTables.js",
-    "compile": "npm run compile:clean && npm run compile:build",
+    "compile": "npm run clean:lib && npm run compile:build",
     "compile:build": "lerna run compile --parallel",
     "compile:clean": "rm -rf packages/**/lib",
```

Comment:
Should this also be removed, since it is a duplicate of `clean:lib`?

Reply:
Removed here: 1867d83b7a899ee4bedaf6a3973a61f36ba6c1ea